broker-daemon/utils/create-http-server.js

```diff
@@ -33,7 +33,7 @@ function handle404 (logger, req, res) {
  * @param {string} opts.logger
  * @returns {ExpressApp}
  */
-function createHttpServer (protoPath, rpcAddress, { disableAuth = false, enableCors = false, privKeyPath, pubKeyPath, logger }) {
+function createHttpServer (protoPath, rpcAddress, { disableAuth = false, enableCors = false, privKeyPath, pubKeyPath, httpMethods = ['*'], logger }) {
```

Comment:
@treygriffith should we just default `httpMethods` at the point of use? We default the value in multiple places but then have a `WILDCARD` var in `grpc-gateway`